icu4c/source/test/intltest/dtifmtts.cpp

```diff
@@ -1673,7 +1673,9 @@ void DateIntervalFormatTest::testFormattedDateInterval() {
         const char16_t* message = u"FormattedDateInterval test 1";
         const char16_t* expectedString = u"July 20 \u2013 25, 2018";
         LocalPointer<Calendar> input1(Calendar::createInstance("en-GB", status));
+	if (status.errIfFailureAndReset()) { return; }
```

Comment:
For several of these, the indentation is off by one. Please fix.

Reply:
Seems like the indentation is tabs, but the rest of the file uses spaces.

Reply:
Tabs would be even worse...

Reply:
Done.